core/templates/dev/head/pages/exploration_player/PlaythroughService.js

```diff
@@ -244,7 +245,8 @@ oppia.factory('PlaythroughService', [
           stateName, destStateName, interactionId, answer, feedback,
           timeSpentInStateSecs) {
         if (this.isPlayerExcludedFromSamplePopulation() ||
-            !this.isExplorationWhitelisted(playthrough.expId)) {
```

Comment:
Let's put these both into a helper function so that this looks like:

```javascript
if (!this.isRecordingPlaythrough(playthrough)) {
  return;
}
```

Reply:
Done.